<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="6" hostname="DESKTOP-J6CSB15" failures="1" tests="1" name="Default test" time="20.724" errors="0" timestamp="2023-01-08T16:39:48 CET">
  <testcase classname="testSuite.PdpPomTestScripts" name="VerifyUserIsAbleToSelectVariationsOfTheProduct" time="6.036">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at base.CommonPOM.verifyByPageSourceContains(CommonPOM.java:109)
at testSuite.PdpPomTestScripts.VerifyUserIsAbleToSelectVariationsOfTheProduct(PdpPomTestScripts.java:171)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- VerifyUserIsAbleToSelectVariationsOfTheProduct -->
  <testcase classname="testSuite.PdpPomTestScripts" name="TC_DETAILS_003_ReviewWithLoggedInUser" time="0.0">
    <ignored/>
  </testcase> <!-- TC_DETAILS_003_ReviewWithLoggedInUser -->
  <testcase classname="testSuite.PdpPomTestScripts" name="VerifyRentalItemCalendarSendKeys" time="0.0">
    <ignored/>
  </testcase> <!-- VerifyRentalItemCalendarSendKeys -->
  <testcase classname="testSuite.PdpPomTestScripts" name="VerifyRentalItemCalendarFORLoop" time="0.0">
    <ignored/>
  </testcase> <!-- VerifyRentalItemCalendarFORLoop -->
  <testcase classname="testSuite.PdpPomTestScripts" name="VerifyRentalItemCalendarLOOP" time="0.0">
    <ignored/>
  </testcase> <!-- VerifyRentalItemCalendarLOOP -->
  <testcase classname="testSuite.PdpPomTestScripts" name="VerifyCorrectProductIsAdded" time="0.0">
    <ignored/>
  </testcase> <!-- VerifyCorrectProductIsAdded -->
  <testcase classname="testSuite.PdpPomTestScripts" name="TC_DETAILS_013_VerifyDonwloadSample" time="0.0">
    <ignored/>
  </testcase> <!-- TC_DETAILS_013_VerifyDonwloadSample -->
</testsuite> <!-- Default test -->
